src/components/Login/LogoutAction.tsx

```diff
@@ -0,0 +1,35 @@
+import { AxiosError } from 'axios'
+import { useRouter } from 'next/router'
+import { useState, useEffect, DependencyList } from 'react'
+
+import { ory } from '../../lib/orysdk'
+
+export function CreateLogoutHandler(deps?: DependencyList) {
```

Comment:
The logic is right and your skills are improving very fast. Congrats.

But this way the handlers creates the flow every time the sidebar is rendered, shouldn't we execute the whole logic only when the menu is clicked? That way we avoid unnecessary calls to Ory.